webhooks/invitation_webhook.go

```diff
@@ -33,9 +36,14 @@ func (v *InvitationValidator) Handle(ctx context.Context, req admission.Request)
 	log.V(1).WithValues("invitation", inv).Info("Validating")
 
 	username := req.UserInfo.Username
+	if !strings.HasPrefix(username, v.UsernamePrefix) {
+		return admission.Denied(fmt.Sprintf("no username prefix expected prefix %q, current user %q", v.UsernamePrefix, username))
```

Comment:
The message looks a bit mangled, does my suggestion retain the correct information?

```suggestion
		return admission.Denied(fmt.Sprintf("Invalid username: only usernames with prefix %q are accepted, got username %q", v.UsernamePrefix, username))
```